HISTORY.rst

```diff
@@ -2,6 +2,20 @@
 History
 =======
 
+1.2.0 (2020-06-05)
+------------------
+
+* Changed public interface of the learners to be more in line with the
+  scikit-learn interface (ongoing).
+* Rewrite and document normalized discounted cumulative gain (ndcg) metric.
+* Fix passing fit keyword arguments on to the core network in
+  ``FATEChoiceFunction``.
+* Fix arguments for ``AllPositive`` baseline.
+* Raise ValueError rather than silently using a default value for unknown
+  passed arguments.
+* Implement linting using flake8 and fix various raised warnings.
```

Comment:
Maybe we should be a bit more general here, since `flake8` is far from the only change
```suggestion
* Internal efforts to increase code quality and make use of linting (`black`, `flake8`, `doc8`).
```